spring-boot-project/spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/json/JsonTestersAutoConfiguration.java

```diff
@@ -234,11 +235,16 @@ static class AbstractJsonMarshalTesterRuntimeHints implements RuntimeHintsRegist
 
 		@Override
 		public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
+			Method initializeMethod = ReflectionUtils.findMethod(this.tester, "initialize", Class.class,
+					ResolvableType.class);
+
+			if (initializeMethod == null) {
+				throw new IllegalStateException("unable to find initialize method for " + this.tester);
+			}
+
 			ReflectionHints reflection = hints.reflection();
 			reflection.registerType(this.tester, MemberCategory.INVOKE_DECLARED_CONSTRUCTORS);
-			reflection.registerMethod(
-					ReflectionUtils.findMethod(this.tester, "initialize", Class.class, ResolvableType.class),
-					ExecutableMode.INVOKE);
+			reflection.registerMethod(initializeMethod, ExecutableMode.INVOKE);
```

Comment:
I'd rather this failed with an NPE as the initialize method should always be present.